app/controls/form/views/select.ejs

```diff
@@ -13,6 +13,15 @@
     >
         <%
             var phValue, phText;
+
+            /*
+             *  This is a fix for falsey placeholder values such as '0' which are converted to 0
+             *  and, as a result, display the conditional display check for placeholders on line 25..
+            */
+            if(typeof this.placeholder === 'number') {
```

Comment:
Aha yeah the $.data() call converts data from string. It's a shame. I don't think we should be re-assigning this.placeholder here though. Can we convert it to a string in form.js?